src/southbridge/amd/sb700/sm.c

```diff
@@ -498,7 +498,7 @@ static void sb700_sm_set_resources(struct device *dev)
 	 * as soon as possible.
 	 */
 	byte = pci_read_config8(dev, SB_MMIO_CFG_REG);
```

Comment:
This has been moved to bootblock in the other PR.

Reply:
AFAICT there is no need to do this in bootblock.

Reply:
If we would like to use amdblocks/acpimmio it would be needed. But ok, let's not mess with it here.